.travis.yml

```diff
@@ -5,30 +5,48 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
-  - nightly
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.cache/pip
+    - vendor
```

Comment:
Caches:

* pip global vendors
* composer global vendor
* project vendor

Project vendor would be updated but just the necessary.

This Travis config was adopted and patched on many sonata project.

cc @rande